project/plugins.sbt

```diff
@@ -1,5 +1,3 @@
-addSbtPlugin("com.dwijnand"      % "sbt-dynver"   % "4.1.1")
-addSbtPlugin("com.dwijnand"      % "sbt-travisci" % "1.2.0")
-addSbtPlugin("de.heikoseeberger" % "sbt-header"   % "5.6.0")
-addSbtPlugin("org.foundweekends" % "sbt-bintray"  % "0.6.1")
-addSbtPlugin("org.scalameta"     % "sbt-scalafmt" % "2.4.6")
+addSbtPlugin("com.dwijnand"      % "sbt-ci-release" % "1.5.10")
+addSbtPlugin("de.heikoseeberger" % "sbt-header"     % "5.6.0")
```

Comment:
Why "com.dwijnand"? The "official" group ID, according to https://github.com/sbt/sbt-ci-release, is "com.github.sbt".

Reply:
My bad, I didn't change the org.